Source/WebCore/page/Quirks.h

```diff
@@ -110,7 +111,7 @@ class Quirks {
     WEBCORE_EXPORT static void updateStorageAccessUserAgentStringQuirks(HashMap<RegistrableDomain, String>&&);
     WEBCORE_EXPORT String storageAccessUserAgentStringQuirkForDomain(const URL&);
     WEBCORE_EXPORT static bool needsIPadMiniUserAgent(const URL&);
-    WEBCORE_EXPORT static bool needsIPhoneUserAgent(const URL&);
+    WEBCORE_EXPORT static bool needsIPhoneUserAgent(API::WebsitePolicies&, const URL&);
```

Comment:
It doesn't seem like you need `policies` here at all.

Reply:
ha yes.